README.md

```diff
@@ -191,6 +191,62 @@ This option can also be set using `-alertmanager.uris` flag. Example:
 
 This variable is required and there is no default value.
 
+#### ANNOTATIONS_DEFAULT_HIDDEN
```

Comment:
Probably, but that would be an ugly UI if I dump it under current settings menu, so I'm not even going to try right now.